provider/baked/.gitattributes

```diff
@@ -0,0 +1,3 @@
+**/* linguist-generated=true
```

Comment:
Observation: @Manishearth had suggested putting .rs.data in gitattributes for syntax highlighting

Reply:
It is in the top-level `.gitattributes` because there are a few baked files outside of `provider/baked`